source/fonts.css

```diff
@@ -1,76 +1,54 @@
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/');
-	src: local('Proxima Nova'), local('ProximaNovaT-Thin'),
-		url('./fonts/?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNovaT-Thin.woff') format('woff'),
-		url('./fonts/ProximaNovaT-Thin.ttf') format('truetype');
-	font-weight: 100;
-	font-style: normal;
-}
 
 @font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Light.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Light'),
-		url('./fonts/ProximaNova-Light.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Light.woff') format('woff'),
-		url('./fonts/ProximaNova-Light.ttf') format('truetype');
-	font-weight: 300;
-	font-style: normal;
+    font-family: 'creato';
+     src: url('./fonts/');
+    src: url('./fonts/creatodisplay-light-webfont.eot');
+    src: local('creato'), local('creatodisplay-light-webfont'),
+         url('./fonts/creatodisplay-light-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-light-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-light-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-light-webfont.ttf') format('truetype');
+    font-weight: 300;
+    font-style: normal;
 }
 
 @font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Regular.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Regular'),
-		url('./fonts/ProximaNova-Regular.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Regular.woff') format('woff'),
-		url('./fonts/ProximaNova-Regular.ttf') format('truetype');
-	font-weight: normal;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-black-webfont.eot');
+    src: local('creato'), local('creatodisplay-black-webfont'),
+         url('./fonts/creatodisplay-black-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-black-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-black-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-black-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
 }
```

Comment:
"Black" fonts should be `font-weight: 900`, not `normal`.

Reply:
Changing it now. Thank you